### PR TITLE
Finished tests and fixed @Autowired

### PR DESCRIPTION
**Pull request by rocketer13579**

### CODE REVIEW

File: src/main/java/com/revature/byteshare/ingredient/IngredientController.java [hunk 1]
```diff
@@ -1,12 +1,32 @@
...
 
-@Controller
+import java.util.List;
+
+@RestController
+@RequestMapping("/ingredients")
 public class IngredientController {
     private final IngredientService ingredientService;
 
     public IngredientController(IngredientService ingredientService) {
```
Comment (aramsing) [inline src/main/java/com/revature/byteshare/ingredient/IngredientController.java:14]: Autowired Annotation should be here

File: src/main/java/com/revature/byteshare/ingredient/IngredientService.java [hunk 2]
```diff
@@ -1,12 +1,32 @@
...
+import com.revature.byteshare.util.exceptions.NutritionixException;
 import org.springframework.stereotype.Service;
 
+import java.util.List;
+
 @Service
 public class IngredientService {
     public final IngredientRepository ingredientRepository;
 
     public IngredientService(IngredientRepository ingredientRepository) {
```
Comment (aramsing) [inline src/main/java/com/revature/byteshare/ingredient/IngredientService.java:13]: Autowired Annotation should be here

**Review by aramsing — CHANGES_REQUESTED** [review]
Apart from where you need Autowired, everything else looks good, Ethan, are there any other changes

**Review by aramsing — APPROVED** [review]
Looks good, merge if you think it looks good Ethan

**Review by env7295 — APPROVED** [review]

**Pull request merged**